src/Resources/views/macros/data-field-type.html.twig

```diff
@@ -392,19 +394,19 @@
 		<ul class="nav nav-tabs">
 			{% for value in values|default([]) %}
 				<li class="{% if loop.index == 1%}active{% endif %}">
-					<a href="#item_{{ dataField.fieldType.id~'_'~loop.index }}__tab" data-toggle="tab" aria-expanded="{% if loop.index == 1%}true{% else %}false{% endif %}">
+					<a href="#item_{{ path|join('_')~'_'~loop.index0 }}__tab" data-toggle="tab" aria-expanded="{% if loop.index == 1%}true{% else %}false{% endif %}">
```

Comment:
index0 ?

Reply:
Good! Yes I learned something

loop.index0 | The current iteration of the loop starting at 0.